packages/stores/types.ts

```diff
@@ -535,6 +537,8 @@ type TUiStore = {
               ]
             | []
     ) => void;
+    setMT5MigrationModalEnabled: (value: boolean) => void;
```

Comment:
do you think `setMT5MigrationModal(value)` or `setMT5MigrationModalStatus(value)` would be a better name, since we are passing enable/disable argument value?
you may also ignore this.